axum/src/docs/extract.md

```diff
@@ -499,6 +499,67 @@ let app = Router::new().route("/foo", get(handler));
 # };
 ```
 
+## Cannot implement both `FromRequest` and `FromRequestParts`
+
+Note that you will make your extractor unusable by implementing both `FromRequest` and `FromRequestParts` directly for the
+same type, unless it is wrapping another extractor:
```

Comment:
Now you need to re-wrap this 😛 

Also how about mentioning the blanket impl of `FromRequest` here? I guess it could be confusing to people how they're allowed to implement both at all then?

Reply:
> Also how about mentioning the blanket impl of FromRequest here? I guess it could be confusing to people how they're allowed to implement both at all then?

Hm how would you phrase that?

Reply:
I'm not sure. Maybe not that important? The whole handler thing is probably just "magic" to most users anyways 😅